keyboards/ergodash/rev1/keymaps/tw1t611/keymap.c

```diff
@@ -0,0 +1,27 @@
+#include QMK_KEYBOARD_H
+#include "keymap_german.h"
+
+#define _QWERTZ 0
+#define _MOD 1
+
+enum custom_keycodes {
+  _QWERTZ = SAFE_RANGE,
+  _MOD,
+};
```

Comment:
This entire block should be replaced with just:
```suggestion
enum layers {
  _QWERTZ,
  _MOD,
};
```

Specifically the custom keycodes is creating compiler issues due to redefining the same C Macros

Reply:
k, ty for reply.